cloud/performance.rst

```diff
@@ -46,20 +46,22 @@ Scaling Out (adding new nodes)
 
 Node addition completes in five to ten minutes, which is faster than node resizing because the new nodes are created without data. To take advantage of the new nodes you still must adjust manually rebalance the shards, meaning move some shards from existing nodes to the new ones.
 
-Citus does not automatically rebalance on node creation because shard rebalancing takes locks on rows whose shards are being moved, degrading write performance for other database clients. The slowdown isn't terribly severe because Citus moves data a shard (or a group of colocated shards) at a time while inserts to other shards can continue normally. However, for maximum control, the choice of when to run the shard rebalancer is left to the database administrator.
```

Comment:
Can we also add a link to our main rebalancer section: https://docs.citusdata.com/en/v7.1/admin_guide/cluster_management.html#rebalancing-shards-without-downtime, so people can learn more about this?

Reply:
(Added a link, just in a slightly different place)